base/linalg/uniformscaling.jl

```diff
@@ -178,6 +178,11 @@ function copy!(A::AbstractMatrix, J::UniformScaling)
     return A
 end
 
+function cond{T}(J::UniformScaling{T})
+    onereal = inv(one(real(J.λ)))
```

Comment:
Why `inv`?  What is wrong with returning an `Int` for `UniformScaling{Int}`, for example?